### PR TITLE
Add command-line args to create-account script for easy automation.

### PR DESCRIPTION
**Pull request by LyonesGamer**

### CODE REVIEW

**Comment by ElijahCaine** [comment]
Let's add a `-h` `--help` flag to document this before merging.
I'll review this tomorrow once it's got a bit more polish.

**Comment by Kennric** [comment]
approving this with the proviso that intermittent bugs have to be addressed soon. +1

File: scripts/create-account.js [hunk 1]
```diff
@@ -32,71 +35,105 @@ function onErr(err) {
   return 1;
 }
 
+if (yargs.argv.h || yargs.argv.help) {
```
Comment (ElijahCaine) [inline scripts/create-account.js]: Does `yargs` not have an automatic help menu like [python's argparse library](https://docs.python.org/3.5/library/argparse.html)?

**Comment by ElijahCaine** [comment]
Where are my usage docs? (in addition to -h flags the README or our usage docs should be updated)

File: scripts/create-account.js (hunk 1 above)
Comment (LyonesGamer) [inline scripts/create-account.js]: It would appear not, no. It just leaves that to the user.

**Comment by ElijahCaine** [comment]
@LyonesGamer Following the README:
```
(tristan/create-account-args)$ npm install
[...]
(tristan/create-account-args)$ npm run create-account -u root -p root_pass
npm ERR! Error: ENOENT, open '/path/to/timesync-node/node_modules/create-account/package.json'
npm ERR! If you need help, you may report this *entire* log,
npm ERR! including the npm and node versions, at:
npm ERR!     <http://github.com/npm/npm/issues>

npm ERR! System Linux 3.16.0-4-amd64
npm ERR! command "/usr/bin/nodejs" "/usr/bin/npm" "run" "create-account" "-u" "root" "-p" "root_pass"
npm ERR! cwd /path/to/timesync-node
npm ERR! node -v v0.10.29
npm ERR! npm -v 1.4.21
npm ERR! path /path/to/timesync-node/node_modules/create-account/package.json
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /path/to/timesync-node/npm-debug.log
npm ERR! not ok code 0
```

**Comment by LyonesGamer** [comment]
That's a slight typo I fixed in the latest commit, it should be `npm run create-account **--** -u USER -p PASSWORD`. With the `--` it works properly, otherwise it passes `-u USER -p PASSWORD` to npm instead of the script.

**Comment by ElijahCaine** [comment]
@LyonesGamer I tried it with the -- and got the same error iirc. I'll re-install dependencies and try again.

**Pull request merged**